pkg/agents/team6/config.go

```diff
@@ -97,7 +97,7 @@ func (a *CustomAgent6) Run() {
 	a.Log("Team 6 took:", infra.Fields{"foodTaken": foodAmount, "bType": a.currBehaviour.String()})
 	a.Log("Team 6 agent has HP:", infra.Fields{"hp": a.HP()})
 
-	msg := messages.NewResponseMessage(a.Floor(), true)
+	msg := messages.NewAskHPMessage(a.Floor())
```

Comment:
Why are you changing this???

Reply:
because responseMessage can not be sent without a requestMessage.

Reply:
...that's a good point

Reply:
Pinging @MattScottEEE  since this is your team's code.

Reply:
Totally fine - we were just playing around with message types so this code doesn't convey any real meaning outside of MVP